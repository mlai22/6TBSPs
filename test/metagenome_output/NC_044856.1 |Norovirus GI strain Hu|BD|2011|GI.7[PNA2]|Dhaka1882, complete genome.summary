NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009724389.1@1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	20	35.03750596048832
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009724389.1@1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	63.19365534788899
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1@4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	63.19365534788899
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009724390.1@1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009724395.1@1-121 ORF7a protein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725295.1@1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1@819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1@4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725309.1 3'-to-5' exonuclease [polyprotein_range=YP_009724389.1@5926-6452] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725310.1 endoRNAse [polyprotein_range=YP_009724389.1@6453-6798] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1@819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009724389.1@1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	695.1302088267788
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725295.1@1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	695.1302088267788
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1@819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	695.1302088267788
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1@819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	695.1302088267788
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009724389.1@1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	754.7761048498853
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009724397.2@1-419 nucleocapsid phosphoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	754.7761048498853
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725295.1@1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	754.7761048498853
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1@819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	754.7761048498853
NC_044856.1 |Norovirus GI strain Hu/BD/2011/GI.7[PNA2]/Dhaka1882, complete genome	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1@819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	754.7761048498853
