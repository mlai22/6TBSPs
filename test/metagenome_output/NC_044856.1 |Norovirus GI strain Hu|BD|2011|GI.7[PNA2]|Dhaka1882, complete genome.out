Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 20
E-value: 35.03750596048832

0     VNDDG  4   
      |*|||      
3699  VYDDG  3703

0     VNDDGV  5   
      |*|||*      
3699  VYDDGA  3704


Frame: -1
Query: KTN
Length: 3
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 16
E-value: 63.19365534788899

0     KTN  2   
      |||      
4441  KTN  4443


Frame: -1
Query: KTN
Length: 3
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 16
E-value: 63.19365534788899

0   KTN  2 
    |||    
49  KTN  51


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1273

Score: 16
E-value: 126.38731069577798

3   DGV  5 
    |||    
87  DGV  89


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009724395.1:1-121 ORF7a protein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 121

Score: 16
E-value: 126.38731069577798

3   DGV  5 
    |||    
68  DGV  70


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 16
E-value: 126.38731069577798

3     DGV  5   
      |||      
1884  DGV  1886


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 126.38731069577798

3     DGV  5   
      |||      
1066  DGV  1068


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 16
E-value: 126.38731069577798

3    DGV  5  
     |||     
335  DGV  337


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009725309.1 3'-to-5' exonuclease [polyprotein_range=YP_009724389.1:5926-6452] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 527

Score: 16
E-value: 126.38731069577798

3    DGV  5  
     |||     
378  DGV  380


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009725310.1 endoRNAse [polyprotein_range=YP_009724389.1:6453-6798] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 346

Score: 16
E-value: 126.38731069577798

3   DGV  5 
    |||    
35  DGV  37

3    DGV  5  
     |||     
182  DGV  184


Frame: 1
Query: VNDDGV
Length: 6
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 126.38731069577798

3     DGV  5   
      |||      
1066  DGV  1068


Frame: -2
Query: KPIKIANNEYDAHIILQKFTSYHYLRIKEGARC
Length: 33
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 16
E-value: 695.1302088267788

0     KPI  2   
      |||      
1987  KPI  1989


Frame: -2
Query: KPIKIANNEYDAHIILQKFTSYHYLRIKEGARC
Length: 33
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 16
E-value: 695.1302088267788

0     KPI  2   
      |||      
1987  KPI  1989


Frame: -2
Query: KPIKIANNEYDAHIILQKFTSYHYLRIKEGARC
Length: 33
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 695.1302088267788

0     KPI  2   
      |||      
1169  KPI  1171


Frame: -2
Query: KPIKIANNEYDAHIILQKFTSYHYLRIKEGARC
Length: 33
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 695.1302088267788

0     KPI  2   
      |||      
1169  KPI  1171


Frame: -3
Query: NQLKLPIMNMMPTLYYKNSLPTIICE
Length: 26
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 15
E-value: 754.7761048498853

0     NQL  2   
      |||      
1941  NQL  1943


Frame: -3
Query: NQLKLPIMNMMPTLYYKNSLPTIICE
Length: 26
Subject: YP_009724397.2:1-419 nucleocapsid phosphoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 419

Score: 15
E-value: 754.7761048498853

0    NQL  2  
     |||     
227  NQL  229


Frame: -3
Query: NQLKLPIMNMMPTLYYKNSLPTIICE
Length: 26
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 15
E-value: 754.7761048498853

0     NQL  2   
      |||      
1941  NQL  1943


Frame: -3
Query: NQLKLPIMNMMPTLYYKNSLPTIICE
Length: 26
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 15
E-value: 754.7761048498853

0     NQL  2   
      |||      
1123  NQL  1125


Frame: -3
Query: NQLKLPIMNMMPTLYYKNSLPTIICE
Length: 26
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 15
E-value: 754.7761048498853

0     NQL  2   
      |||      
1123  NQL  1125


